Hunting Queries/CustomLogs/AWSBucketAPILogs-SuspiciousDataAccesstoS3Buckets.yaml

```diff
@@ -1,8 +1,11 @@
 id: 669e1338-b1a2-4d73-b720-a1e60d5d1474
 name: Suspicious Data Access to S3 Bucket from Unknown IP
 description: |
-  Adversaries may access data objects from improperly secured cloud storage. This query will look for any access originating from Source IP which was not seen historically accessing the bucket or downloading files from it.
-  You can also limit the query to only private buckets with sensitive files.
+  Adversaries may access data objects from improperly secured cloud storage. This query will identify any access originating from a Source IP which was not seen historically accessing the bucket or downloading files from it.
```

Comment:
needs single quotes around just in case characters get updated

Reply:
updated